etc/testing/deploy/aws.sh

```diff
@@ -57,7 +63,11 @@ set -x
 case "${OP}" in
   create)
     aws_sh="$(realpath "$(dirname "${0}")/../../deploy/aws.sh")"
-    sudo "${aws_sh}" --region=${REGION} --zone=${ZONE} --state=${STATE_STORE} --no-metrics
+    cmd=("${aws_sh}" --region=${REGION} --zone=${ZONE} --state=${STATE_STORE} --no-metrics)
+    if [[ -n "${CLOUDFRONT}" ]]; then
+      cmd+=(" ${CLOUDFRONT}")
```

Comment:
I believe the leading space is unnecessary. The way bash arrays work is the parsing is done before the array is created, so by appending this to the array using this syntax you don't need a separator (as opposed to e.g. `cmd="${cmd[*]} ${CLOUDFRONT}"`)